core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -138,6 +140,12 @@ object ConsoleProducer {
                                     .withOptionalArg()
                                     .describedAs("compression-codec")
                                     .ofType(classOf[String])
+    val batchSizeOpt = parser.accepts("batch-size", "Number of messages to send in a single batch if they are not being sent synchronously. "+
+       "please note that this opt will be replaced if max-partition-memory-bytes is also set")
+      .withRequiredArg
+      .describedAs("size")
+      .ofType(classOf[java.lang.Integer])
+      .defaultsTo(200)
```

Comment:
We should make the default value the same the `max-partition-memory-bytes`, which is `16 * 1024`, so that we can do this:
`this opt will be replaced if max-partition-memory-bytes is also set`